### DISCUSSION TITLE
Is there a way to have in output the feed URL?

### DISCUSSION THREAD

**Discussion by aborruso** [adbar/trafilatura#719] [Q&A] [answered]
Hi,
the `feed` command is very convenient, thank you very much.
Is there any way to have not the content or a list of URLs, but to have the main URL of the feed?

Thank you

**Comment by adbar** [comment, marked as answer]
Not as it stands, you could run the software in debugging mode and/or tweak the code to do it.

**Reply by aborruso** [reply]
> Not as it stands, you could run the software in debugging mode and/or tweak the code to do it.

Thank you very much
